### ISSUE TITLE
Placeholder for Entities/DTOs Needed for Model 

### ISSUE THREAD

**Issue by JorjaComer** [US-EPA-CAMD/PI-1#288]
- [ ] Create each model (entity)

AC: Model (s) will exist in repo 

NOTE: this ticket will be plait into a ticket for each entity needed after #287 is complete. will also ad tickets for creating DTOs for each entity, as needed.

**Comment by JorjaComer** [comment]
Split into 3 new tickets. No longer needed.
